### ISSUE TITLE
New submissions for Tue, 14 Feb 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#34]
### Visual Ensemble Analysis of Fluid Flow in Porous Media across Simulation  Codes and Experiment
 - **Authors:** Ruben Bauer, Quynh Quang Ngo, Guido Reina, Steffen Frey, Bernd Flemisch, Helwig Hauser, Thomas Ertl, Michael Sedlmair
 - **Subjects:** Human-Computer Interaction (cs.HC); Graphics (cs.GR)
 - **Arxiv link:** https://arxiv.org/abs/2302.05447
 - **Pdf link:** https://arxiv.org/pdf/2302.05447
 - **Abstract**
 We study the question of how visual analysis can support the comparison of spatio-temporal ensemble data of liquid and gas flow in porous media. To this end, we focus on a case study, in which nine different research groups concurrently simulated the process of injecting CO2 into the subsurface. We explore different data aggregation and interactive visualization approaches to compare and analyze these nine simulations. In terms of data aggregation, one key component is the choice of similarity metrics that define the relation between the different simulations. We test different metrics and find that a fine-tuned machine-learning based metric provides the best visualization results. Based on that, we propose different visualization methods. For overviewing the data, we use dimensionality reduction methods that allow us to plot and compare the different simulations in a scatterplot. To show details about the spatio-temporal data of each individual simulation, we employ a space-time cube volume rendering. We use the resulting interactive, multi-view visual analysis tool to explore the nine simulations and also to compare them to data from experimental setups. Our main findings include new insights into ranking of simulation results with respect to experimental data, and the development of gravity fingers in simulations.
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imagingremote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
### Removing Image Artifacts From Scratched Lens Protectors
 - **Authors:** Yufei Wang, Renjie Wan, Wenhan Yang, Bihan Wen, Lap-pui Chau, Alex C. Kot
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2302.05746
 - **Pdf link:** https://arxiv.org/pdf/2302.05746
 - **Abstract**
 A protector is placed in front of the camera lens for mobile devices to avoid damage, while the protector itself can be easily scratched accidentally, especially for plastic ones. The artifacts appear in a wide variety of patterns, making it difficult to see through them clearly. Removing image artifacts from the scratched lens protector is inherently challenging due to the occasional flare artifacts and the co-occurring interference within mixed artifacts. Though different methods have been proposed for some specific distortions, they seldom consider such inherent challenges. In our work, we consider the inherent challenges in a unified framework with two cooperative modules, which facilitate the performance boost of each other. We also collect a new dataset from the real world to facilitate training and evaluation purposes. The experimental results demonstrate that our method outperforms the baselines qualitatively and quantitatively. The code and datasets will be released after acceptance.
### Combined Location Online Weather Data: Easy-to-use Targeted Weather  Analysis for Agriculture
 - **Authors:** Darren Yates, Christopher Blanchard, Allister Clarke, Sabih-Ur Rehman, Md Zahidul Islam, Russell Ford, Rob Walsh
 - **Subjects:** Social and Information Networks (cs.SI)
 - **Arxiv link:** https://arxiv.org/abs/2302.06142
 - **Pdf link:** https://arxiv.org/pdf/2302.06142
 - **Abstract**
 The continuing effects of climate change require farmers and growers to have greater understanding of how these changes affect crop production. However, while climatic data is generally available to help provide much of that understanding, it can often be in a form not easy to digest. The proposed Combined Location Online Weather Data (CLOWD) framework is an easy-to-use online platform for analysing recent and historical weather data of any location within Australia at the click of a map. CLOWD requires no programming skills and operates in any HTML5 web browser on PC and mobile devices. It enables comparison between current and previous growing seasons over a range of environmental parameters, and can create a plain-English PDF report for offline use, using natural language generation (NLG). This paper details the platform, the design decisions taken and outlines how farmers and growers can use CLOWD to better understand current growing conditions. Prototypes of CLOWD are now online for PCs and smartphones.
## Keyword: video retrieval
There is no result 
## Keyword: transfer functionmobile
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: volume data
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl2
There is no result 
### Visual Ensemble Analysis of Fluid Flow in Porous Media across Simulation  Codes and Experiment
 - **Authors:** Ruben Bauer, Quynh Quang Ngo, Guido Reina, Steffen Frey, Bernd Flemisch, Helwig Hauser, Thomas Ertl, Michael Sedlmair
 - **Subjects:** Human-Computer Interaction (cs.HC); Graphics (cs.GR)
 - **Arxiv link:** https://arxiv.org/abs/2302.05447
 - **Pdf link:** https://arxiv.org/pdf/2302.05447
 - **Abstract**
 We study the question of how visual analysis can support the comparison of spatio-temporal ensemble data of liquid and gas flow in porous media. To this end, we focus on a case study, in which nine different research groups concurrently simulated the process of injecting CO2 into the subsurface. We explore different data aggregation and interactive visualization approaches to compare and analyze these nine simulations. In terms of data aggregation, one key component is the choice of similarity metrics that define the relation between the different simulations. We test different metrics and find that a fine-tuned machine-learning based metric provides the best visualization results. Based on that, we propose different visualization methods. For overviewing the data, we use dimensionality reduction methods that allow us to plot and compare the different simulations in a scatterplot. To show details about the spatio-temporal data of each individual simulation, we employ a space-time cube volume rendering. We use the resulting interactive, multi-view visual analysis tool to explore the nine simulations and also to compare them to data from experimental setups. Our main findings include new insights into ranking of simulation results with respect to experimental data, and the development of gravity fingers in simulations.
### RAFaRe: Learning Robust and Accurate Non-parametric 3D Face  Reconstruction from Pseudo 2D&3D Pairs
 - **Authors:** Longwei Guo, Hao Zhu, Yuanxun Lu, Menghua Wu, Xun Cao
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2302.05486
 - **Pdf link:** https://arxiv.org/pdf/2302.05486
 - **Abstract**
 We propose a robust and accurate non-parametric method for single-view 3D face reconstruction (SVFR). While tremendous efforts have been devoted to parametric SVFR, a visible gap still lies between the result 3D shape and the ground truth. We believe there are two major obstacles: 1) the representation of the parametric model is limited to a certain face database; 2) 2D images and 3D shapes in the fitted datasets are distinctly misaligned. To resolve these issues, a large-scale pseudo 2D\&3D dataset is created by first rendering the detailed 3D faces, then swapping the face in the wild images with the rendered face. These pseudo 2D&3D pairs are created from publicly available datasets which eliminate the gaps between 2D and 3D data while covering diverse appearances, poses, scenes, and illumination. We further propose a non-parametric scheme to learn a well-generalized SVFR model from the created dataset, and the proposed hierarchical signed distance function turns out to be effective in predicting middle-scale and small-scale 3D facial geometry. Our model outperforms previous methods on FaceScape-wild/lab and MICC benchmarks and is well generalized to various appearances, poses, expressions, and in-the-wild environments. The code is released at this http URL .
### 3D Colored Shape Reconstruction from a Single RGB Image through  Diffusion
 - **Authors:** Bo Li, Xiaolin Wei, Fengwei Chen, Bin Liu
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2302.05573
 - **Pdf link:** https://arxiv.org/pdf/2302.05573
 - **Abstract**
 We propose a novel 3d colored shape reconstruction method from a single RGB image through diffusion model. Diffusion models have shown great development potentials for high-quality 3D shape generation. However, most existing work based on diffusion models only focus on geometric shape generation, they cannot either accomplish 3D reconstruction from a single image, or produce 3D geometric shape with color information. In this work, we propose to reconstruct a 3D colored shape from a single RGB image through a novel conditional diffusion model. The reverse process of the proposed diffusion model is consisted of three modules, shape prediction module, color prediction module and NeRF-like rendering module. In shape prediction module, the reference RGB image is first encoded into a high-level shape feature and then the shape feature is utilized as a condition to predict the reverse geometric noise in diffusion model. Then the color of each 3D point updated in shape prediction module is predicted by color prediction module. Finally, a NeRF-like rendering module is designed to render the colored point cloud predicted by the former two modules to 2D image space to guide the training conditioned only on a reference image. As far as the authors know, the proposed method is the first diffusion model for 3D colored shape reconstruction from a single RGB image. Experimental results demonstrate that the proposed method achieves competitive performance on colored 3D shape reconstruction, and the ablation study validates the positive role of the color prediction module in improving the reconstruction quality of 3D geometric point cloud.
### Denoising and Prompt-Tuning for Multi-Behavior Recommendation
 - **Authors:** Chi Zhang, Rui Chen, Xiangyu Zhao, Qilong Han, Li Li
 - **Subjects:** Information Retrieval (cs.IR)
 - **Arxiv link:** https://arxiv.org/abs/2302.05862
 - **Pdf link:** https://arxiv.org/pdf/2302.05862
 - **Abstract**
 In practical recommendation scenarios, users often interact with items under multi-typed behaviors (e.g., click, add-to-cart, and purchase). Traditional collaborative filtering techniques typically assume that users only have a single type of behavior with items, making it insufficient to utilize complex collaborative signals to learn informative representations and infer actual user preferences. Consequently, some pioneer studies explore modeling multi-behavior heterogeneity to learn better representations and boost the performance of recommendations for a target behavior. However, a large number of auxiliary behaviors (i.e., click and add-to-cart) could introduce irrelevant information to recommenders, which could mislead the target behavior (i.e., purchase) recommendation, rendering two critical challenges: (i) denoising auxiliary behaviors and (ii) bridging the semantic gap between auxiliary and target behaviors. Motivated by the above observation, we propose a novel framework-Denoising and Prompt-Tuning (DPT) with a three-stage learning paradigm to solve the aforementioned challenges. In particular, DPT is equipped with a pattern-enhanced graph encoder in the first stage to learn complex patterns as prior knowledge in a data-driven manner to guide learning informative representation and pinpointing reliable noise for subsequent stages. Accordingly, we adopt different lightweight tuning approaches with effectiveness and efficiency in the following stages to further attenuate the influence of noise and alleviate the semantic gap among multi-typed behaviors. Extensive experiments on two real-world datasets demonstrate the superiority of DPT over a wide range of state-of-the-art methods. The implementation code is available online at https://github.com/zc-97/DPT.
### Fast Real-Time Shading for Polygonal Hair
 - **Authors:** Martin Gerard
 - **Subjects:** Graphics (cs.GR)
 - **Arxiv link:** https://arxiv.org/abs/2302.06468
 - **Pdf link:** https://arxiv.org/pdf/2302.06468
 - **Abstract**
 Though a lot of improvement has been made to hair rendering techniques in the recent years, realistic rendering of hair remains a challenge, especially in real time. In this paper, we propose a fast technique to approximate the shading of hair lighted by an environment map, direct lighting or a global illumination system, without having to render deep opacity maps or requiring additional artistic work.
